### PR TITLE
add page service

### PR DESCRIPTION
**Pull request by bakaichiq**

### CODE REVIEW

File: css/service.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+ul {
+	padding: 0;
+	margin: 0;
+}
+
+li {
+	list-style-type: none; 
+}
+
+div, h1, h2, h3, h4, h5, h6 {
```
Comment (ibokg) [inline css/service.css]: Why is this?

File: css/service.css [hunk 2]
```diff
@@ -0,0 +1,122 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
```
Comment (ibokg) [inline css/service.css]: Include the fonts using the `<link>` tag in html.
It is preferable.

File: css/service.css [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+li {
+	list-style-type: none; 
+}
+
+div, h1, h2, h3, h4, h5, h6 {
+	cursor: default;
+}
+
+body {
+	width: 100%;
```
Comment (ibokg) [inline css/service.css]: I think it is not necessary

File: css/service.css [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+	margin: 0;
+	left: 0;
```
Comment (ibokg) [inline css/service.css]: I think it is not necessary

File: css/service.css [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+	top: 0;
```
Comment (ibokg) [inline css/service.css]: I think it is not necessary

File: css/service.css [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+	font-family: 'Roboto';
+}
+
+.wrap {
+	width: 1180px;
```
Comment (ibokg) [inline css/service.css]: This class is usually called a `container`.
To make the container responsive use this rules:
`.container { width: 100%; max-width: 1180px; margin: 0 auto; }`

File: css/service.css [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+	margin: 0 auto;
+}
+
+/*Главная страница*/
+
+/* Header */
+
+.header {
+	height: 84px;
```
Comment (ibokg) [inline css/service.css]: Try not to set a fixed `height` for the blocks.
This is not a good practice.

File: css/service.css [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+
+.main-content__block {
+	width: 100%;
+	line-height: 60px;
+	border-top: 1px solid #7D7D7D;
+	border-bottom: 1px solid #7D7D7D;
+}
+
+.main-content__title {
+	margin: 0;
```
Comment (ibokg) [inline css/service.css]: Why is this?

File: css/service.css (hunk 1 above)
Comment (bakaichiq) [reply css/service.css]: so that the cursor does not change when hovering over text

File: css/service.css (hunk 8 above)
Comment (bakaichiq) [reply css/service.css]: fixed

File: css/service.css (hunk 6 above)
Comment (bakaichiq) [reply css/service.css]: fixed

File: css/service.css (hunk 2 above)
Comment (bakaichiq) [reply css/service.css]: fixed

File: css/service.css (hunk 1 above)
Comment (ibokg) [reply css/service.css]: @bakaichiq I understood. But why? This violates the UI(user interface) concept.

Comment (bakaichiq) [reply css/service.css]: @ibokg how to fix?

Comment (ibokg) [reply css/service.css]: @bakaichiq This does not need to be fixed. Just delete and that's it.

Comment (bakaichiq) [reply css/service.css]: i deleted!

**Pull request merged**